backend/user.js

```diff
@@ -191,47 +208,161 @@ router.get('/myprofile', authenticateToken, async (req, res) => {
   }
 });
 
-//Delete Patient Profile
+// Delete Patient Profile
 router.delete('/myprofile', authenticateToken, async (req, res) => {
-  try{
+  try {
+    // Find the patient profile to be deleted
+    const patient = await prisma.patient.findUnique({
```

Comment:
it looks like we repeat this code a few times to find the patient - could we put this in a helper function?

Reply:
Agreed, it would be good to also have a helper function that is in charge of creating the notif & sending it via websocket given a message, source user ID, and target user ID